valhalla/baldr/graphtile.h

```diff
@@ -494,7 +497,15 @@ class GraphTile {
       flow_sources = &temp_sources;
     *flow_sources = kNoFlowMask;
 
-    // TODO: current with coefficient based on distance in time from start of route
+    // TODO(danpat): this needs to consider the time - we should not use live speeds if
+    //               the request is not for "now", or we're some X % along the route
+    // TODO(danpat): for short-ish durations along the route, we should fade live
+    //               speeds into any historic/predictive/average value we'd normally use
+    auto directed_edge_index = std::distance(const_cast<const DirectedEdge*>(directededges_), de);
+    auto live_speed = traffic_tile.getTrafficForDirectedEdge(directed_edge_index);
+    if (live_speed.speed_kmh != 0) {
```

Comment:
we should have a special value for closed roads as well and translate that to `0` kmh here right? we shuld also only check this if this is a route with `kCurrentFlowMask` enabled as is done for the other sources below

Reply:
I added both of these pieces of logic.  For now, "closed" means "speed = 0" and "congestion > 4" - defining the congestion values is TBD for now.